tests/auth_tests/test_templates.py

```diff
@@ -53,6 +53,19 @@ def test_PasswordResetConfirmView_valid_token(self):
         self.assertContains(response, '<title>Enter new password</title>')
         self.assertContains(response, '<h1>Enter new password</h1>')
 
+    def test_PasswordResetConfirmView_invalid_uid(self):
+        """
+        ValidationError in user lookups are handled correctly (#29206).
+        """
+        client = PasswordResetConfirmClient()
+        default_token_generator = PasswordResetTokenGenerator()
+        token = default_token_generator.make_token(self.user)
+        uidb64 = urlsafe_base64_encode('NOT_A_VALID_STRING'.encode()).decode()
```

Comment:
Sorry. Could this say `"NOT_A_VALID_UUID"`? I think that would be clearer.

Reply:
I'm wondering, since it's not strictly related to uuids as you correctly pointed out, but to uid in general, if uid would be more appropriate here?

Reply:
But it's only because it's not a valid UUID that the ValidationError is raised, which (that it's then caught) is what the test is checking. 

Hang on... you broke the test...

Reply:
Will re-add the custom user model tomorrow then👌